clients/nutclient.h

```diff
@@ -294,6 +312,18 @@ class Client
 	virtual void deviceMaster(const std::string& dev)throw(NutException)=0;
 	virtual void deviceForcedShutdown(const std::string& dev)throw(NutException)=0;
 
+	/**
+	 * Redeem an async action result from an async ID.
```

Comment:
not sure what you wanna mean by "redeem", but should be reworded.
Also async ID is to be worded "tracking ID", as elsewhere.